tests/foreman/ui/test_errata.py

```diff
@@ -148,6 +152,31 @@ def rhva_vm(module_rhva_repos_col):
         yield client
 
 
+@fixture(scope='module')
+def module_erratatype_repos_col(module_org, module_lce):
+    repos_collection = RepositoryCollection(
+        distro=DISTRO_RHEL7,
+        repositories=[
+            SatelliteToolsRepository(),
+            # As Satellite Tools may be added as custom repo and to have a "Fully entitled" host,
+            # force the host to consume an RH product with adding a cdn repo.
+            RHELAnsibleEngineRepository(cdn=True),
+            # add a repo with errata of different types (security, advisory, etc)
+            YumRepository(url=FAKE_9_YUM_REPO),
+        ]
+    )
+    repos_collection.setup_content(module_org.id, module_lce.id)
+    return repos_collection
+
+
+@fixture
+def erratatype_vm(module_erratatype_repos_col):
```

Comment:
Since this is scoped per test, and there is only one test using this, does it have to be fixture? Can you just create VM inside test?

Reply:
Yeah I kinda did it this way in case there might be future tests that need this setup.